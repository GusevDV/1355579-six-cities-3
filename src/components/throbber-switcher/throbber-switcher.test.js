import React from "react";
import renderer from "react-test-renderer";
import ThrobberSwitcher from "./throbber-switcher.jsx";

it(`Should Throbber component render`, () => {

  const render = renderer.create(
      <ThrobberSwitcher isLoading={true} render={()=>(
        <div>Test</div>
      )} />
  )
  .toJSON();

  expect(render).toMatchSnapshot();
});

it(`Should loaded render`, () => {

  const render = renderer.create(
      <ThrobberSwitcher isLoading={false} render={()=>(
        <div>Loaded</div>
      )}/>
  )
  .toJSON();

  expect(render).toMatchSnapshot();
});
