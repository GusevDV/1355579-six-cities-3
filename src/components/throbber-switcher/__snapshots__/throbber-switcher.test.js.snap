// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Throbber component render 1`] = `
<div
  className="throbber "
>
  <img
    alt="throbber"
    className="throbber__img"
    height={100}
    src="/img/throbber.svg"
    width={100}
  />
</div>
`;

exports[`Should loaded render 1`] = `
<div>
  Loaded
</div>
`;
